server/src/main/java/org/opensearch/search/profile/query/QueryProfiler.java

```diff
@@ -81,14 +81,14 @@ public void startRewriteTime() {
     /**
      * Stop recording the current rewrite and add it's time to the total tally, returning the
      * cumulative time so far.
-     *
-     * @return cumulative rewrite time
      */
-    public long stopAndAddRewriteTime() {
-        return ((AbstractQueryProfileTree) profileTree).stopAndAddRewriteTime();
+    public void stopAndAddRewriteTime() {
```

Comment:
Why signature change here? `AbstractQueryProfileTree.stopAndAddRewriteTime` still returns long

Reply:
Hey @reta, `QueryProfiler.stopAndAddRewriteTime()` is only called by `ContextIndexSearcher.rewrite()`, and we don't need the return any value for that.

Reply:
Thanks @reta for the review. It's addressed now.